packages/core-app-api/src/app/AppManager.tsx

```diff
@@ -310,8 +310,14 @@ export class AppManager implements BackstageApp {
 
           // Go through the featureFlags returned from the traversal and
           // register those now the configApi has been loaded
+          const registeredFlags = featureFlagsApi.getRegisteredFlags();
```

Comment:
Probably worth creating a set of these for simpler lookup in the loop: `new Set(registered.flags.map(f => f.name))`

Reply:
Thanks!